eZ/Publish/Core/Limitation/NewSectionLimitationType.php

```diff
@@ -165,4 +158,45 @@ public function valueSchema()
     {
         throw new \eZ\Publish\API\Repository\Exceptions\NotImplementedException(__METHOD__);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCriterionByTarget(APILimitationValue $value, APIUserReference $currentUser, ?array $target): CriterionInterface
+    {
+        if (empty($target)) {
```

Comment:
hmm, the name of the argument is target, but type hint implies it's plural like `evaluate`